templates/website/pol-info-survey.html

```diff
@@ -0,0 +1,21 @@
+<?php	
+	
+$values['title'] = "Help us by answering a few quick questions";	
+$values['robots'] = 'noindex, nofollow';	
+	
+template_draw('header', $values);	
+	
+?>
+
+<script type="text/javascript" >
+document.write('<script src="http' + ( ("https:" == document.location.protocol) ? "s" : "") + 
```

Comment:
Always https so this can be simplified.

Reply:
Also, shouldn't be a `document.write`, but just a normal script element, please.